Adds a new outfit to the autodrobe

War0100 opened the pull request:
basically this adds a new set of three outfits to the autodrobe, the outfits are replica Nuclear Operative hardsuits. However, they're sprited in such a way so you can easily distinguish the two apart.
On line 189 it says that the bee shoes were removed, however I only moved "//yogs clothes for autodrobe end here" and replaced the ")" with a "," to make it work properly.

War0100 committed:
Adds a new outfit to the autodrobe

This change adds a new outfit to the autodrobe, a set of three Nuclear Operative replica hardsuits. Colored to be lighter than the original so you can distinguish the two.

War0100 committed:
Updated Misc.dm

Updated Misc.dm

Amelia0010 reviewed (changes requested):
As well, the worn sprite (not the sprite of them item when it's in your hand, but the sprite of the item when you're wearing it) needs to be added to:

Helmet: https://github.com/yogstation13/Yogstation-TG/blob/master/icons/mob/head.dmi
Suit: https://github.com/yogstation13/Yogstation-TG/blob/master/icons/mob/suit.dmi

War0100 committed:
Update miscellaneous.dm

War0100 committed:
Update miscellaneous.dm

War0100 committed:
Update miscellaneous.dm

War0100 committed:
misc suits and misc head fix

my small brained ass fucked up and added the helmet to the suit misc.dm, it's now in head misc.dm

Amelia0010 reviewed (dismissed):
the worn sprite (not the sprite of them item when it's in your hand, but the sprite of the item when you're wearing it) needs to be added to:

Helmet: https://github.com/yogstation13/Yogstation-TG/blob/master/icons/mob/head.dmi
Suit: https://github.com/yogstation13/Yogstation-TG/blob/master/icons/mob/suit.dmi

War0100 committed:
added in sprites I forgot

added the worn sprites (couldnt find them at first and wasnt sure on how it worked) to head.dmi and suit.dmi including their proper directions

Amelia0010 committed:
Merge branch 'master' into autodrobeoutfit

alexkar598 reviewed (changes requested):
Issues in icons/mob/head.dmi:

Unused sprites: hardsuit1-syndi - S
Unused sprites: hardsuit1-syndi - W
Unused sprites: hardsuit1-syndi - E
Unused sprites: hardsuit1-syndi - N
Unused sprites: hardhat1_atmos -
Deleted sprites: hardhat1_atmos
Deleted sprites: hardhat1_syndi
Unneeded animation: fake_nuke_helmet_
Wrong name: fake_nuke_helmet_

Issues in icons/mob/suit.dmi:
Wrong name: fake_nuke_suit_
